mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -106,6 +106,18 @@ private[ml] trait HasProbabilityCol extends Params {
   def getProbabilityCol: String = get(probabilityCol)
 }
 
+private[ml] trait HasFitIntercept extends Params {
+  /**
+   * param for fitting the intercept term
+   * @group param
+   */
+  val fitIntercept: BooleanParam =
```

Comment:
Thinking more about this, can you please make this default to true?  (Add "Some(true)" as a 4th argument, plus update doc.)  Thanks!